tools/gen-device-svd.py

```diff
@@ -311,15 +324,17 @@ def parseRegister(groupName, regEl, baseAddress, bitfieldPrefix=''):
                     'elementsize': reg.size(),
                 })
             # set first result bitfield
-            shortName = reg.name().replace('_%s', '').replace('%s', '')
+            shortName = reg.name().replace('_%s', '').replace('%s', '').upper()
             results[0]['bitfields'] = parseBitfields(groupName, shortName, fieldsEls, bitfieldPrefix)
             return results
-
+    regName = reg.name()
+    if not regName[0].isupper() and not regName[0].isdigit():
+        regName = regName.upper()
```

Comment:
I haven't seen the code, but wouldn't it lead to more readable code if just the first character is capitalized?

```python
def capitalizeFirstChar(s):
    return s[:1].upper() + s[1:]
```

Reply:
This is to keep as much consistency as possible since most SVD files use all caps. The HiFive SVD files use all lower case for some (not all) elements.